subcell_pipeline/analysis/compression_metrics/_compare_compression_metrics.py

```diff
@@ -0,0 +1,132 @@
+# %% [markdown]
+# # Compare metrics across simulators
+
+# %% [markdown]
+"""
+Notebook contains steps to compare metrics of fiber compression across different
+simulators. Currently supports comparison of Cytosim and ReaDDy simulations.
+
+- [Load dataframes](#load-dataframes)
+- [Calculate metrics and add to dataframe](#calculate-metrics)
+- [Plot metrics vs time](#plot-metrics-vs-time)
+"""
```

Comment:
Fix TOC links

Reply:
Fixed in [384b16](https://github.com/simularium/subcell-pipeline/pull/58/commits/384b16b4ca7645013baeb7b217276296ce9d9128)